src/app/helptext/sharing/webdav/webdav.ts

```diff
@@ -26,12 +26,14 @@ export const helptext_sharing_webdav = {
   tooltip_enabled: T('Enable this WebDAV share. Unset to disable this WebDAV \
  share without deleting it.'),
 
-  column_name: T('Share Name'),
+  column_name: T('Name'),
   column_comment: T('Description'),
   column_path: T('Path'),
   column_enabled: T('Enabled'),
   column_ro: T('Read Only'),
-  column_perm: T('Change User and Group Ownership'),
+  column_perm: T('Ownership'),
+
+  column_perm_tooltip: T('Change User and Group Ownership'),
```

Comment:
Now that the checkbox is disabled, the wording here should change from `Change User and Group Ownership` to just `User and Group ownership`

Reply:
Nice catch. Updated